Libraries/plist/CMakeLists.txt

```diff
@@ -69,6 +69,11 @@ add_executable(plutil Tools/plutil.cpp)
 target_link_libraries(plutil plist process util)
 install(TARGETS plutil DESTINATION usr/bin)
 
+add_executable(PlistBuddy Tools/PlistBuddy.cpp ${CMAKE_SOURCE_DIR}/ThirdParty/linenoise/linenoise.c)
+target_include_directories(PlistBuddy PRIVATE "${CMAKE_SOURCE_DIR}/ThirdParty/linenoise")
+target_link_libraries(PlistBuddy plist util)
+install(TARGETS PlistBuddy DESTINATION usr/bin)
```

Comment:
Would it work to make it optional? Check if `${CMAKE_SOURCE_DIR}/ThirdParty/linenoise` exists, and compile it in if it does. Can do the same check in the code with `__has_include(<linenoise.h>)` (or pass it in as a `#define` from CMake).